crates/router/src/core/refunds.rs

```diff
@@ -665,9 +665,19 @@ pub async fn refund_list(
         .map(ForeignInto::foreign_into)
         .collect();
 
+    let total_count = db
+        .get_total_count_of_refunds(
+            &merchant_account.merchant_id,
+            &req,
+            merchant_account.storage_scheme,
+        )
+        .await
+        .to_not_found_response(errors::ApiErrorResponse::InternalServerError)?;
```

Comment:
Shouldn't this be 0 instead of error response?

Reply:
count 0 case is handled, error can occur in cases, such as DB is down or some unexpected behaviour.